phreaknet.1.md

```diff
@@ -57,6 +57,9 @@ the best Asterisk and DAHDI experience.
 **install**
 : Install or upgrade Asterisk. This is the primary command provided by PhreakScript.
 
+**source**
+: Download source code only, without building or installing. This operates on the current working directory.

```

Comment:
This makes it sound like only the original source is downloaded but not patched. I'd add that it is patched, unless --vanilla is use